### DISCUSSION TITLE
Drawing attention to the methods paper

### DISCUSSION THREAD

**Discussion by lzachmann** [lzachmann/models-for-missing-data#4] [Ideas]
Wondering if any one has led the development of an I&M article, like this: https://www.nps.gov/articles/monitoring-in-the-context-of-climate-change.htm? (There's quite a few I&M related articles, as you can see with [this Google search](https://www.google.com/search?q=national+park+service+I%26M+articles&sxsrf=AOaemvJUc8r_36Gh7SNpY7cAlgh9jalfhQ%3A1631732582406&ei=ZkNCYfeiGJ-lqtsPnLCbiAg&oq=national+park+service+I%26M+articles&gs_lcp=Cgdnd3Mtd2l6EAMyBwgjELADECcyBwgAEEcQsAMyBwgAEEcQsAMyBwgAEEcQsAMyBwgAEEcQsANKBAhBGABQAFgAYMI9aANwAngAgAEAiAEAkgEAmAEAyAEFwAEB&sclient=gws-wiz&ved=0ahUKEwj37cTH1YHzAhWfkmoFHRzYBoEQ4dUDCA8&uact=5)). I'll bet we can craft something that is, at least, not totally off-topic. 

I feel like the biggest case we need to make concerns the concept of ignorability. Without getting into any details on the models or their implementation, it may still be worth saying something -- at a high level of abstraction -- about what we mean by nonignorability, about how complex monitoring designs require appropriate models, and about the risks of failing to use appropriate models. Getting it right is more than merely 'academic'. Real-world consequences of ignoring the unignorable include bias, overconfidence, misrepresentation of the ecology / process, drawing the wrong conclusions or making the wrong management interventions by attributing change to the wrong drivers / variables, etc., etc. Would it make sense to try to outline the basic talking points here? Thoughts?

**Comment by megcswan** [comment]
I think maybe Carolyn could see if Alice might have time to write up a web article on the paper? Or Erin could you check with Sonya?

**Reply by lzachmann** [reply]
Thanks Megan, @clivensperger and @eborgman what do you think?

**Reply by eborgman** [reply]
I worked with Sonya and a bunch of others on a web article about alpine monitoring. [https://www.nps.gov/subjects/mountains/gloria.htm]. Regardless of whether Sonya or Alice helps out (both are great choices!), we'd have to take the first stab at the content. Drafting out the talking points - perhaps in an outline format - sounds good. Sonya likes starting with outlines anyway. 

I can ask what Sonya's availability is. What is our timeline? There isn't a horrible rush, right? Our paper isn't going anywhere. Sonya is often booked out several weeks to a month in advance.

**Reply by lzachmann** [reply]
Great, thanks Erin. If we all feel this is a good idea, maybe we ping Sonya relatively soon just to reserve a block of time on her calendar? And yes! I don't think there's a giant rush. The paper isn't going anywhere, and having a month or two between now and then would give us enough time to create a useful, well-crafted outline so that Sonya or Alice can hit the ground running.

**Reply by clivensperger** [reply]
Yep, I think a web article seems like a good format for this. I can check with Alice as well just to see if she has interest/availability, in case one of them seems much busier than the other!

**Reply by eborgman** [reply]
Sonya says she would have time later next week! That's sooner than I thought. Tomorrow I might start thinking about an outline and creating a shared doc for us to use.

**Reply by clivensperger** [reply]
Oh good, well I didn't say anything to Alice yet so I will hold off on that ; ) I'm off tomorrow and Monday but happy to help review something when I'm back.

**Reply by eborgman** [reply]
@lzachmann Does git have some way to have people concurrently contributing to a shared document? I was going to use microsoft onedrive (the google doc equivalent), but I can't share it with you. (This is for the outline)

**Reply by lzachmann** [reply]
Does Google Docs work for you all? https://docs.google.com/document/d/1qEzSPER00JXs-DS0udC-j36q2_fKoMwXVegbDMgtFZo/edit?usp=sharing

**Reply by eborgman** [reply]
Yep, that'll work. Thanks! @clivensperger @megcswan @cllmcintyre Everyone: the draft outline will be in the google doc above. Feel free to edit! (We should get Dana access too - I'll see if I can share it with her)

**Comment by lzachmann** [comment]
@eborgman was your plan to focus on case studies 1 & 3 in the web article?

**Reply by eborgman** [reply]
Currently all 3 are in there, with only 1 type of missing data discussed for each. I was thinking we could see how long the article was getting and have people weigh in on how many case studies we should stick with. I think Sonya's close to getting a draft out to everyone to review, based on [a version 4 outline draft](https://teams.microsoft.com/l/file/4084E5B9-8DCC-442E-9A50-41139E1F26C8?tenantId=0693b5ba-4b18-4d7b-9341-f32f400a5494&fileType=docx&objectUrl=https%3A%2F%2Fdoimspp.sharepoint.com%2Fsites%2Fext-nps-upland-bayesian-trends%2FShared%20Documents%2FGeneral%2FWeb%20article%2FOutline-draft%20text%20v4.docx&baseUrl=https%3A%2F%2Fdoimspp.sharepoint.com%2Fsites%2Fext-nps-upland-bayesian-trends&serviceName=teams&threadId=19:86d0a802359c4e71a980ded02112492d@thread.skype&groupId=4d21ee58-9a66-49e4-808d-af292d146a45).
